spec/system/candidate_interface/candidate_signs_in_without_account_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate tries to sign in without an account' do
+  scenario 'Candidate signs in and recieves an email inviting them to sign up' do
+    given_the_pilot_is_open
+
+    given_i_am_a_candidate_without_an_account
```

Comment:
Maybe:

```
given_the_pilot_is_open
and_i_am_a_candidate_without_an_account
```

Feels weird that we have to do `given_the_pilot_is_open`. Is there any particular reason we have to do it for this system spec and not others?

Reply:
When the pilot is closed, it's not possible to sign up, proceed from start page and the apply from find page. All other things, like signing in and applying, are still possible.